src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAwaitForeachTests.cs

```diff
@@ -1456,32 +1494,33 @@ public System.Threading.Tasks.Task<bool> MoveNextAsync()
         }
 
         [Fact]
-        public void TestPatternBased_MissingCancellationToken()
+        public void TestPatternBased_GetEnumeratorWithoutCancellationToken()
```

Comment:
>TestPatternBased_GetEnumeratorWithoutCancellationToken [](start = 20, length = 54)

Is there a test with `GetEnumerator(CancellationToken token)` with non-optional `CancellationToken`?